keyboards/neopad/readme.md

```diff
@@ -0,0 +1,25 @@
+# Neopad
+
+Neopad is a little macropad with four switches and two rotary encoders, made by through hole components only.
+_Actually the number of switches is six, because even the encoder are allowed to click._
+
+![Neopad](https://i.imgur.com/TgOkj2Fh.jpg "Neopad first proto")
+
+The Neopad in the photo above is the first prototype. See the project repository for revision 1 update and KiCad files.
+
+* Keyboard maintainer: [rookiebwoy](https://github.com/rookiebwoy)
+* Hardware supported: ProMicro, _Elite-C (not tested)_
+* Project repository: [Neopad on github](https://github.com/rookiebwoy/neopad)
+
+Make example for Neopad (after setting up your build environment):
+
+    make neopad/rev1:default
+
+Install examples:
+
+    make neopad/rev1:default:avrdude
```

Comment:
```suggestion
Make example for this keyboard (after setting up your build environment):

    make neopad/rev1:default

Flashing example for this keyboard:

    make neopad/rev1:default:flash
```

Reply:
Ok for the grammar changes.
Not really sure about changing :avrdude to :flash because even in the [official documentation](https://beta.docs.qmk.fm/using-qmk/guides/flashing/flashing) under "Caterina" subsection they suggest to use :avrdude for Arduino clones. Can I keep it?

Reply:
`:flash` takes into account the `BOOTLOADER` set in rules.mk and runs the appropriate command. It's much easier than remembering or trying to find which bootloader a board uses. For this reason it is part of the [readme template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).

Reply:
Ok, tested and it works. Thank you for teach me all this things :)